tests/e2e/setup/setup.go

```diff
@@ -159,16 +156,6 @@ func (s *E2ETestSuite) initGenesis() {
 	appGenState, genDoc, err := genutiltypes.GenesisStateFromGenFile(genFilePath)
 	s.Require().NoError(err)
 
-	// Gravity Bridge
-	var gravityGenState gravitytypes.GenesisState
-	s.Require().NoError(s.cdc.UnmarshalJSON(appGenState[gravitytypes.ModuleName], &gravityGenState))
-
-	gravityGenState.Params.BridgeChainId = uint64(EthChainID)
-
-	bz, err := s.cdc.MarshalJSON(&gravityGenState)
```

Comment:
add `:` when create bz in the line 165 `bz, err := s.cdc.MarshalJSON(&bech32GenState)`